src/spikeinterface/core/base.py

```diff
@@ -543,22 +543,26 @@ def load(file_path, base_folder=None):
             # case from a folder after a calling extractor.save(...)
             folder = file_path
             file = None
-            
-            # the is spikeinterface<=0.94.0
-            # a folder came with 'cached.sjon'
-            for dump_ext in ('json', 'pkl', 'pickle'):
-                f = folder / f'cached.{dump_ext}'
+
+            if folder.suffix == ".zarr":
+                from .zarrrecordingextractor import read_zarr
+                extractor = read_zarr(folder)
+            else:
+                # the is spikeinterface<=0.94.0
+                # a folder came with 'cached.json'
+                for dump_ext in ('json', 'pkl', 'pickle'):
+                    f = folder / f'cached.{dump_ext}'
+                    if f.is_file():
+                        file = f
+
+                # spikeinterface>=0.95.0
+                f = folder / f'si_folder.json'
                 if f.is_file():
                     file = f
-            
-            # spikeinterface>=0.95.0
-            f = folder / f'si_folder.json'
-            if f.is_file():
-                file = f
```

Comment:
Do something like:

```
if (folder / f'si_folder.json').is_file():
    from .binaryfolder import read_binary_folder
    extractor = read_binary_folder(folder)
else:
    # back-compatibility
```

Reply:
this is actually needed because a folder can be a `BinaryRecording`, `NpzSorting`, or `NumpySnippets`